dotnet-client-libraries/IntroductionToWolverineLibrary/IntroductionToWolverineLibrary/Program.cs

```diff
@@ -0,0 +1,44 @@
+using IntroductionToWolverineLibrary.Models;
+using Wolverine;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+builder.Services.AddControllers();
+builder.Host.UseWolverine( x =>
+{
+    x.PublishAllMessages().ToLocalQueue("local-queue");
+});
+
+var app = builder.Build();
+
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();
+}
+
+app.UseHttpsRedirection();
+
+app.MapPost("/newOrder", async (NewBookOrder newOrder, IMessageBus bus) => await bus.InvokeAsync(newOrder))
```

Comment:
Instead of "newOrder", the endpoint name can be simply "order". Since it is a post api, it implies we are creating a new order record.

Reply:
Renamed to '/order'.